lib/language_pack/ruby.rb

```diff
@@ -348,6 +372,8 @@ def setup_profiled
       if ruby_version.jruby?
         add_to_profiled set_jvm_max_heap
         add_to_profiled set_java_mem
+        add_to_export set_jvm_stack_size
+        add_to_export set_java_stack_size
```

Comment:
I think these should be `add_to_profiled`. The `export` script is for multi-buildpack support.